git-quick-stats

```diff
@@ -269,7 +280,9 @@ ADDITIONAL USAGE
     You can set _GIT_BRANCH to set the branch of the stats
         ex: export _GIT_BRANCH=master
     You can set _GIT_IGNORE_AUTHORS to filter out specific authors
-        ex: export _GIT_IGNORE_AUTHORS=\"(author1|author2)\""
+        ex: export _GIT_IGNORE_AUTHORS=\"(author1|author2)\"
+    You can sort contribution stats by field name, commits, insertions, deletions, or lines - total lines changed and order - asc, desc
+        ex: export _GIT_SORT_BY=name-asc"
```

Comment:
Very minor finding. Should this be double-quoted in the example? We should be able to escape these i.e. `ex: export _GIT_SORT_BY=\"name-asc\""`

Feel free to ignore if you don't think it's necessary.

IMPORTANT: If you do make this change, then `tests/commands_test.sh` also needs to be tweaked.